src/libAtomVM/term.c

```diff
@@ -407,6 +407,17 @@ static int term_type_to_index(term t)
     }
 }
 
+#define CMP_POP_AND_CONTINUE()                                                                     \
```

Comment:
Should this define be at the top of the module?  I am not sure what the proper convention is here, though I think you have asked for defines to be defined at the top, even if they are only used locally.

Reply:
indeed, I prefer defines and macros at the top, but honestly seeing this as a helper specific to term_compare, seemsa more natural to keep it close to it.